src/components/members.component.js

```diff
@@ -65,10 +71,14 @@ export const MembersList = ({
       showsHorizontalScrollIndicator={false}
       renderItem={({ item }) =>
         <TouchableHighlight
-          onPress={() =>
-            navigation.navigate('Profile', {
-              user: item,
-            })}
+          onPress={() => {
+            navigation.navigate(
+              authUser.login === item.login ? 'AuthProfile' : 'Profile',
```

Comment:
I noticed that you use the short if syntax on 1 line here and on line 255 for example of the 'comment-list-item.component.js' file you spread it over multiple lines. Would be nice if we have a consistency.